db/migrate/20150923125825_add_job_invocation_task_group.rb

```diff
@@ -0,0 +1,7 @@
+class AddJobInvocationTaskGroup < ActiveRecord::Migration
+  def change
+    alter_table :foreman_tasks_task_groups do |t|
+      t.integer :job_invocation_id, index: true
```

Comment:
This association would be better on the `job_invocations` part to prevent the sparse tables